### DISCUSSION TITLE
Using a homodimer as bait for Pulldowns?

### DISCUSSION THREAD

**Discussion by nazzo1** [KosinskiLab/AlphaPulldown#230] [Q&A] [answered]
Hello,
Can anyone please explain how to run a Pulldown test using a homo-dimer protein as the bait? I'm not sure which mode I need to use and how to specify that the bait is a dimer of itself for testing against target monomer proteins.
Thanks for developing this great tool!
-Nazzo

**Comment by jkosinski** [comment]
Yes, that is possible and you need to use custom mode with multimeric templates. https://github.com/KosinskiLab/AlphaPulldown/blob/main/manuals/example_3.md
Note, the feature of using multimeric templates is still very experimental.
@DimaMolod can help you when he's back.

**Reply by DimaMolod** [reply]
I think it depends on what exactly you are trying to achieve. If you have a pdb/mmcif file of the homo-dimer that you want to use as a template, you can use mutlimeric templates as in example 3.
You may also be interested in the default AF modeling with the standard pipeline, then it would be enough to run AP in custom mode using custom.txt file as described in https://github.com/KosinskiLab/AlphaPulldown/blob/main/manuals/example_2.md

**Reply by nazzo1** [reply]
Thanks for the help! 

I essentially just need to run the basic pulldown mode, as outlined in example 1, but with a homo-dimer bait protein. I am testing a protein of unknown function against all the proteins in a bacterial genome (over 4,300 proteins), and have finished the screen with the monomer protein. But the highest ranking protein pulldown target that AlphaPulldown identified was with itself, so now I want to test the homo-dimer. 

If I'm understanding the custom pulldown mode explained in example 2, I don't need separate baits and target .txt files that list the protein names. Instead I can just add the bait protein name twice before the target on every line. So it would look like: 
bait;bait;target1
bait;bait;target2
bait;bait;target3
Etc... Is this right?

Also, I have already computed the MSA of all >4,300 genes in my target bacteria genome using v0.30.7. Will those files be compatible with v1.0? 

Thanks again,
Nazzo

**Reply by DimaMolod** [reply, marked as answer]
@nazzo1 that's right, you can use custom mode with the custom.txt looking like in your example, or you can simply provide bait.txt twice using a coma, for details see https://github.com/KosinskiLab/AlphaPulldown/issues/188
